dbt_common/tests.py

```diff
@@ -0,0 +1,10 @@
+_TEST_CACHING_ENABLED: bool = False
+
+
+def test_caching_enabled() -> bool:
+    return _TEST_CACHING_ENABLED
+
+
+def enable_test_caching() -> None:
```

Comment:
can we add a `disable_test_caching()` method?

Reply:
Sure.